tests/tck/features/optimizer/PushFilterDownGetNbrsRule.feature

```diff
@@ -62,6 +62,35 @@ Feature: Push Filter down GetNeighbors rule
       | Project      | 1            |                                 |
       | GetNeighbors | 2            | filter: (serve.start_year>2005) |
       | Start        |              |                                 |
+    When profiling query:
```

Comment:
Could you add some test cases like this:
```
$var=go from "Tim Duncan" over like yield like._dst as dst,$$.player.age as age;
go from $var.dst over like where like.likeness IN [v IN [1,3] WHERE v+$var.age > 0
```

Reply:
But how to profile a query with a `$var`?